### PR TITLE
Build PowerSlice package compatible with Find 13 

### PR DESCRIPTION
**Pull request by vnbaaij**
Build PowerSlice package compatible with Find 13

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\PowerSlice",
+    "\\PowerSlice\\Properties"
+  ],
+  "SelectedNode": "\\PowerSlice\\Properties\\AssemblyInfo.cs",
+  "PreviewInSolutionExplorer": false
+}
```
Comment (kdelacreme) [inline .vs/VSWorkspaceState.json]: I don't think this file should be added

Comment (kdelacreme) [reply .vs/VSWorkspaceState.json]: I can't put comments on the binaries, but I'm doubtful the .vs/slnx.sqlite should be included?

Comment (vnbaaij) [reply .vs/VSWorkspaceState.json]: I'm going to exclude the whole .vs folder. It contains status info for the specific Visual Studio user that is working on the project and should not be in repo. Will be rebuild when someone opens the solution locally

File: .gitignore [hunk 2]
```diff
@@ -164,3 +164,4 @@ pip-log.txt
 
 # Mac crap
 .DS_Store
+/.vs/**/*
```
Comment (kdelacreme) [inline .gitignore]: So maybe move this upward in the file to the Visual Studio section, just for clarity.

**Pull request merged**